ports/meta/macathena-extra-software-nox/debathena-list

```diff
@@ -0,0 +1,26 @@
+# the remainder of the debathena-extra-software-nox list, that isn't in macathena-extra-software
+Source: debathena-extra-software-nox
+Section: debathena/base
+Priority: extra
+Standards-Version: 3.9.3
+Maintainer: Debathena Project <debathena@mit.edu>
+Build-Depends: cdbs, debhelper, dh-buildinfo
+
+Package: debathena-extra-software-nox
+Architecture: all
+Depends: build-essential, debathena-debian-dev, # I think these are debian specific
+  emacs-intl-fonts, # does this come with emacs?
+  athena-jot, # jot is already present on mac I think? But I don't know if athena-jot is the same thing.
+  ispell, iamerican, # ispell conflicts with aspell, which is also in debathena-extra-software-nox
+  drscheme | plt-scheme, # there are lots of other schemes but I don't see this one
+  inn2-inews, # can't find this
+  perl-doc, # does this come with perl?
```

Comment:
Yes, this comes with perl. Debian is weird and wants their documentation in a separate package (I think for licensing reasons?)